charts/sonarr/README.md

```diff
@@ -1,27 +1,28 @@
-# sonarr televsion show download client
+# Sonarr | Smart PVR for newsgroup and bittorrent users
+Umbrella chart that
+* Uses [media-common](https://github.com/k8s-at-home/charts/tree/master/charts/media-common) as a base
+* Adds docker image information leveraging the [Linuxserver.io image](https://hub.docker.com/r/linuxserver/sonarr/)
+* Deploys [Sonarr](https://github.com/sonarr/Sonarr)
 
-This is a helm chart for [sonarr](https://github.com/sonarr/sonarr/) leveraging the [Linuxserver.io image](https://hub.docker.com/r/linuxserver/sonarr/)
-
-## TL;DR;
-
-```shell
+## TL;DR
+```console
 $ helm repo add k8s-at-home https://k8s-at-home.com/charts/
 $ helm install k8s-at-home/sonarr
 ```
 
 ## Installing the Chart
-
-To install the chart with the release name `my-release`:
-
+To install the chart with the release name `sonarr`:
 ```console
-helm install --name my-release k8s-at-home/sonarr
+helm install sonarr k8s-at-home/sonarr
 ```
 
 ## Upgrading
+Chart versions before 6.0.0 did not use media-common. Upgrading will require you to nest your values.yaml file under
+a top-level `sonarr:` key.
 
-Chart versions 3.2.0 and earlier used separate PVCs for Downloads and TV. This presented an issue where Sonarr would be unable to hard-link files between the /downloads and /tv directories when importing media. This is caused because each PVC is exposed to the pod as a separate filesystem. This resulted in Sonarr copying files rather than linking; using additional storage without the user's knowledge.
+Chart versions 3.2.0 and earlier used separate PVCs for Downloads and Music. This presented an issue where Sonarr would be unable to hard-link files between the /downloads and /music directories when importing media. This is caused because each PVC is exposed to the pod as a separate filesystem. This resulted in Sonarr copying files rather than linking; using additional storage without the user's knowledge.
```

Comment:
Probably related to copy/paste, but I suspect references to 'Music' should really be 'TV'